azure-pipelines.yml

```diff
@@ -300,6 +300,31 @@ jobs:
       msbuildArgs: '/p:AppxPackageSigningEnabled=false'
     displayName: 'Build UWP_x86'
 
+- job: uwp_napi_jsi_x64
+  pool:
+    vmImage: 'windows-latest'
+
+  steps:
+  - script: |
+      git submodule update --init --recursive
+    displayName: 'Checkout dependencies'
+
+  - script: |
+      cd Validation/napi-jsi
+      npm install
```

Comment:
not sure if folks know whether npm is installed by default on windows agents?

Reply:
or if there is a better way to flag this needed dependency

Reply:
Yes, it is preinstalled (it's part of Node).
https://github.com/actions/virtual-environments/blob/main/images/win/Windows2019-Readme.md

Reply:
More generally: https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/hosted?view=azure-devops&tabs=yaml#software

Reply:
sounds good, im going to look into also using the npm install azure devops task